yaml/csr_template.yaml

```diff
@@ -53,7 +55,9 @@
 - csr: misa
   description: >
     Machine ISA Register
+  address: 0x301
   privilege_mode: M
+  read_only: 0
```

Comment:
Should this be applied to each field rather the whole register? BTW, does this mean the "type" is now redundant or inaccurate?

Reply:
good point, there is also this configurability for each field as well, so the CSR level read_only is not needed.